scripts/run_integration_tests_aarch64.sh

```diff
@@ -180,6 +180,12 @@ update_workloads() {
 
 process_common_args "$@"
 
+# aarch64 not supported for MSHV
+if [[ "$hypervisor" = "mshv" ]]; then
+    echo "Aarch64 is not supported in Microsoft Hypervisor"
+    exit 0
```

Comment:
I don't think it should exit 0. It's an error to try and combine msvh with aarch64.

Reply:
Updated.